### PR TITLE
pioneers config and launch files changes

### PR DESCRIPTION
**Pull request by t-constantin**
Pull request to add working controllers and to fix launch files config or URDF settings. 

Using the differential_drive plugin in Gazebo make the robots able to be controlled by sending to the /cmd_vel topic a linear speed value and an agular speed value.
Teleop_twist_keyboard can also use this topic to control the robot from a terminal.

### CODE REVIEW

File: swarmus_ros_description/urdf/pioneer.urdf.xacro [hunk 1]
```diff
@@ -240,64 +239,64 @@
 
   <gazebo>
     <plugin name="gazebo_ros_control" filename="libgazebo_ros_control.so">
-      <robotSimType>gazebo_ros_control/DefaultRobotHWSim</robotSimType>
+    <!--robotSimType>gazebo_ros_control/DefaultRobotHWSim</robotSimType-->
```
Comment (stlaperriere) [inline swarmus_ros_description/urdf/pioneer.urdf.xacro:242]: Utile de garder ce commentaire?

File: swarmus_ros_description/urdf/pioneer.urdf.xacro [hunk 2]
```diff
@@ -240,64 +239,64 @@
...
     </plugin>
   </gazebo>
 
   <gazebo>
     <plugin name="differential_drive" filename="libgazebo_ros_diff_drive.so">
-      <robotNamespace>$(arg prefix)</robotNamespace>
+      <!--robotNamespace>$(arg prefix)</robotNamespace-->
```
Comment (stlaperriere) [inline swarmus_ros_description/urdf/pioneer.urdf.xacro:248]: Utile de garder ce commentaire?

File: swarmus_ros_description/config/pioneer_control.yaml [hunk 3]
```diff
@@ -1,17 +1,14 @@
...
 
+# Position Controllers ---------------------------------------
+right_position_controller:
+  type: effort_controllers/JointVelocityController
+  joint: right_wheel_hinge
+  pid: {p: 1.0, i: 0.01, d: 1.0}
+left_position_controller:
+  type: effort_controllers/JointVelocityController
+  joint: left_wheel_hinge
+  pid: {p: 1.0, i: 0.01, d: 1.0}
```
Comment (t-constantin) [inline swarmus_ros_description/config/pioneer_control.yaml:14]: Unused file, will be deleted

File: swarmus_ros_description/urdf/pioneer.urdf.xacro (hunk 1 above)
Comment (t-constantin) [reply swarmus_ros_description/urdf/pioneer.urdf.xacro:242]: Je ne sais pas, mais on le voit dans plein de tuto. Je vais l'enlever et si jamais c'est utile on le retrouvera facilement

File: swarmus_ros_description/urdf/pioneer.urdf.xacro (hunk 2 above)
Comment (t-constantin) [reply swarmus_ros_description/urdf/pioneer.urdf.xacro:248]: deleted

**Pull request merged**